lib/Sema/TypeCheckConcurrency.cpp

```diff
@@ -1211,6 +1211,9 @@ namespace {
       bool isPotentiallyIsolated = false;
       if (!var) {
         isPotentiallyIsolated = false;
+      } else if (var->getName().str().equals("__secretlyKnownToBeLocal")) {
+        // FIXME(distributed): rdar://84029304 this must be implemented properly
```

Comment:
FWIW, I don't like radar references in code, because only Apple people can interpret them. The FIXME should say what's wrong: we don't have a proper way to model "we did a dynamic check and know that this actor is local, but we can't express that to the type system".

Reply:
Ok! Never sure about style of those (if to use radar or not) happy to add wording instead! 👍

Reply:
// FIXME(distributed): we did a dynamic check and know that this actor is local,
        //  but we can't express that to the type system; the real implementation
        //  will have to mark 'self' as "known to be local" after an is-local check.